.github/dependabot.yml

```diff
@@ -51,7 +50,6 @@ updates:
           - '@types/*'
           - '*eslint*'
           - '*prettier*'
-          - 'tsc-watch'
```

Comment:
Is this also related to transifex? @elwinschmitz

Reply:
Not directly. But it was the last dangling-dependency...
And I thought "_Hmm.. is this even used anywhere?_"... and then I checked and found nothing. ;)